src/Ironclad/Sdk/AuthCookieMiddleware.cs

```diff
@@ -56,5 +65,36 @@ await handler.HandleRequestAsync())
 
             await this.next.Invoke(context);
         }
+
+        private bool UserUsesLatestIos(HttpContext context)
+        {
+            var userAgent = context.Request.Headers["User-Agent"].ToString();
+
+            var rx = new Regex(@"OS ((\d+_?){2,3})\s");
```

Comment:
Regex is never changed, so it's better to move it to the static field with Compiled flag to increase performance.
```suggestion
            private static Regex IosRegex = new Regex(@"OS ((\d+_?){2,3})\s",  RegexOptions.Compiled);
```